### PR TITLE
[Tests-Only] Check for version tags while running web tests

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
- Run full ci for web tests git version tags are included

## Related Issue

- https://github.com/owncloud/ocis/issues/2107

## Motivation and Context
- https://github.com/owncloud/ocis/pull/2116#issuecomment-855462259

## How Has This Been Tested?

- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by phil-davis** [comment]
@kiranparajuli589 we are thinking different things.

We want full-CI triggered automagically when a git tag happens. (Nothing to do with scenario tags)

https://github.com/owncloud/ocis/blob/master/.drone.star#L999

**Comment by micbar** [comment]
Yes, @phil-davis I meant github tags.

File: .drone.star [hunk 1]
```diff
@@ -500,14 +501,14 @@ def uiTests(ctx):
 
     filterTags = params["filterTags"]
 
-    if ("full-ci" in ctx.build.title.lower()):
+    if ("full-ci" in ctx.build.title.lower() or ctx.build.event == "tag"):
```
Comment (phil-davis) [inline .drone.star:504]: This looks like the only change that is required. If the event is a tag, then the full CI will run.
The information that is available about the build is in this object:
https://github.com/drone/drone/blob/master/plugin/converter/starlark/args.go

If one of those items has the tag name in it, then we can also test that the tag name starts with "v". But if not, then it is not really a problem - we can run full CI on every tag (all the tags on `master` should start with "v" anyway)

Comment (kiranparajuli589) [reply .drone.star:504]: yes sure @phil-davis,
also the trigger has some regex like with `v*`:
```
"trigger": {
            "ref": [
                "refs/heads/master",
                "refs/tags/v*",
                "refs/pull/**",
            ],
        },
```,
maybe tat also filters the tag

**Comment by kiranparajuli589** [comment]
```

✖ When the user restores file "data.zip" from the trashbin using the webUI # tests/acceptance/stepDefinitions/filesContext.js:861 | 290s
-- | --
2598 | Cannot read property 'trim' of null | 290s
2599 | TypeError: Cannot read property 'trim' of null

```
https://drone.owncloud.com/owncloud/ocis/5331/43/6

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/5332/43/6
```
runsh: Total unexpected failed scenarios throughout the test run:
webUITrashbinRestore/trashbinRestore.feature:14
```

I will add that to the "intermittent fails" issue. And restart CI.

**Comment by kiranparajuli589** [comment]
> https://drone.owncloud.com/owncloud/ocis/5332/43/6
> 
> ```
> runsh: Total unexpected failed scenarios throughout the test run:
> webUITrashbinRestore/trashbinRestore.feature:14
> ```
> 
> I added that to the "intermittent fails" issue [owncloud/web#4919 (comment)](https://github.com/owncloud/web/issues/4919#issuecomment-856434313) - And restarted CI.

this was passing yesterday, and passing locally too.. 😢 
I've used a web PR to log the result, whose value gets null.

**Comment by kiranparajuli589** [comment]
waiting for https://github.com/owncloud/web/pull/5205

**Comment by phil-davis** [comment]
> waiting for [owncloud/web#5205](https://github.com/owncloud/web/pull/5205)

@kiranparajuli589 I merged the web PR. Please update the web master commit id here, and CI should be great again...

**Review by phil-davis — APPROVED** [review]

**Pull request merged**